app/controllers/barcode_items_controller.rb

```diff
@@ -1,12 +1,15 @@
 class BarcodeItemsController < ApplicationController
   def index
-    @barcode_items = current_organization.barcode_items.includes(:item).filter(filter_params)
+    @barcode_items = BarcodeItem.includes(:item).where(organization_id: current_organization.id).filter(filter_params)
```

Comment:
Why the change for the index action? It "seems" to be the same query.

Reply:
TBH I don't remember why I did this change... I may have done it before I corrected the association scoping?